Frends.FTP.ListFiles/Frends.FTP.ListFiles/Definitions/ListObject.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Frends.FTP.ListFiles.Definitions
+{
+    /// <summary>
+    /// Single file data.
+    /// </summary>
+    public class ListObject
+    {
+        /// <summary>
+        /// Last modified timestamp of the file. 
+        /// </summary>
+        /// <example>3.5.2022 0.00.00</example>
```

Comment:
Can we have a better example, please. Lets put it in ISO 8601, which is standard internationally (e.g. 2022-05-20T13:21:42Z).